Source/Plugins/Core/com.equella.core/js/__stories__/components/StarRating.stories.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to The Apereo Foundation under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * The Apereo Foundation licenses this file to you under the Apache License,
+ * Version 2.0, (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Meta, Story } from "@storybook/react";
+import * as React from "react";
+import { StarRating, StarRatingProps } from "../../tsrc/components/StarRating";
+
+export default {
+  title: "StarRating",
```

Comment:
Sorry, should of mentioned this before. Bit by bit I'm trying to keep these stories a bit more structured in the Storybook UI. As a result, could you please change this to:

```suggestion
  title: "component/StarRating",
```